experiments/standalone/Vegetation/no_vegetation.jl

```diff
@@ -253,8 +253,8 @@ axislegend(ax)
 save(joinpath(savedir, "no_veg_state.png"), fig)
 fig2 = Figure()
 ax = Axis(fig2[1, 1], xlabel = "Time (days)", ylabel = "Energy Fluxes")
-lines!(ax, sol.t ./ 24 ./ 3600, SW_abs, label = "SW")
-lines!(ax, sol.t ./ 24 ./ 3600, LW_abs, label = "LW")
+lines!(ax, sol.t ./ 24 ./ 3600, SW_n, label = "SW")
```

Comment:
Usually, we don't calculate SW_n or LW_n. We keep all fluxes separate, i.e., SW_d, SW_u, LW_d, LW_u, or we already compute Rn.

Reply:
Here I am checking them individually just because I want to make sure each behaves correctly as LAI -> 0